app/assets/javascripts/settings.js

```diff
@@ -8,3 +8,25 @@ $(document).ready(function() {
     }
   });
 });
+
+function test_mail(item, user_id, url) {
+  $(item).addClass("disabled");
+  $('#test_indicator').show();
+  var param = {id: user_id};
+  $.ajax({
+    url: url,
+    type: 'put',
+    data: param,
+    success: function(result, textstatus, xhr) {
+      notify("<p>" + result.message + "</p>", 'success');
+    },
+    error: function (xhr) {
+      var error = $.parseJSON(xhr.responseText).message;
+      notify("<p>" + error + "</p>", 'error');
```

Comment:
This doesn't seem to render correctly:

![screenshot from 2016-11-11 15-17-10](https://cloud.githubusercontent.com/assets/482089/20219580/05f441b2-a822-11e6-9e70-000502daea69.png)